samples/sample_common/include/current_date.h

```diff
@@ -1,5 +1,5 @@
 /******************************************************************************\
-Copyright (c) 2005-2017, Intel Corporation
+Copyright (c) 2005-2018, Intel Corporation
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```

Comment:
Do we still use '#define PRODUCT_COPYRIGHT' in this file below? If yes - please, update it from 2014 to 2018. If we don't use it - delete.

Reply:
This is standard header for all samples code. I don't see any PRODUCT_COPYRIGHT in this change

Reply:
@fzhar : It is not changed in your change and that's a problem. Look here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/0eeeccf96cd0f7f867a17152c8a44cc6a0893a08/samples/sample_common/include/current_date.h#L25